CONTRIBUTING.rst

```diff
@@ -240,6 +240,7 @@ package:
 Maintainer tasks
 ================
 
+
```

Comment:
This isn't really needed?

Reply:
Needed...
```
 validate-pyproject git:(fix-linkcheck) ✗ reading sources... [100%] readme .. schemas

CONTRIBUTING.rst:243: CRITICAL: Title level inconsistent:

Releases
--------
```

Reply:
So that is how to fix this warning!

The rst spec is surprisingly bit pedantic O.o

Reply:
Wow, nice find. 😮